webpack.config.js

```diff
@@ -4,13 +4,13 @@ const webpack = require('webpack')
 const determineHost = () => {
   switch ( process.env.NODE_ENV ) {
     case 'production':
-      return "'http://localhost:1337'"
+      return 'http://localhost:1337'
```

Comment:
Ok it turns out that you are right, we don't need the extra "". Keep it this way.